testdata/p4_16_samples_outputs/issue1806-first.p4

```diff
@@ -1,6 +1,7 @@
 #include <core.p4>
-#define V1MODEL_VERSION 20180101
```

Comment:
I admit I am a bit fuzzy on this V1MODEL_VERSION variable that was used here, and do not know how to check whether removing it is a good idea or a bad idea in the test programs where you have removed it.   Any advice you might provide on helping guide my thinking about when this V1MODEL_VERSION is supposed to be used, vs. not?

Reply:
I believe this variable is added automatically by toP4 [here](https://github.com/p4lang/p4c/blob/main/frontends/p4/toP4/toP4.cpp#L178) whenever `v1model.p4` is included.